package.json

```diff
@@ -39,5 +39,5 @@
   "engines": {
     "node": ">=20.0.0"
   },
-  "packageManager": "pnpm@8.8.0"
-}
\ No newline at end of file
+  "packageManager": "pnpm@8.8.0+sha512.798e6b322669ce63c8da856bd62ad1f7b6f36f4dfaa0ac025af2bdd700d09dd5dca943e5cad3edac5d1b3baebc4b2c3fb80fbb8537f936733c32be2ec7805144"
+}
```

Comment:
@orangecoloured I think we don't need to specify sha here

Reply:
It was some kind of a mess up. Fixed now.